core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -3536,6 +3541,24 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 HANDLE_OP_END();
             }
 
+#if WASM_ENABLE_SIMDE != 0
```

Comment:
`#if WASM_ENABLE_SIMDE != 0` this should be `#if WASM_ENABLE_SIMD != 0` but I'll fix it once we make the spec tests work and are ready to launch

Reply:
I do think we might not need WASM_ENABLE_SIMDE flag in the first place, it probably should be set based on fast-interp + simd